packages/backend/src/workers/trigger.ts

```diff
@@ -55,6 +55,17 @@ worker.on('failed', (job, err) => {
   )
 })
 
+worker.on('error', (err) => {
+  if (!err) {
+    logger.error('Worker undefined error')
+    return
+  }
+  // catch-all just in case any errors bubble up and potentially crash the worker task
+  logger.error(`Worker errored with ${err.message}`, {
+    err: err.stack,
+  })
+})
```

Comment:
same paranoia

```suggestion
try {
  if (!err) {
    logger.error('Worker undefined error')
    return
  }
  // catch-all just in case any errors bubble up and potentially crash the worker task
  logger.error(`Worker errored with ${err.message}`, {
    err: err.stack,
  })
  }  catch (e) {
    // eslint-disable-next-line no-console
    console.log('Error happened to logger in worker')
    console.error(e)
  }
})
```

Reply:
As discussed, not needed